src/Uno.UI/UI/Xaml/Controls/Primitives/ButtonBase/ButtonBase.cs

```diff
@@ -177,12 +177,6 @@ protected override void OnPointerReleased(PointerRoutedEventArgs args)
 		{
```

Comment:
`PointerPressed` and `PointerReleased` should be marked as handled (cf. https://github.com/nventive/Uno/pull/865 ) Not sure about other pointer events